app/components/blacklight/document/bookmark_component.rb

```diff
@@ -21,6 +21,13 @@ def bookmarked?
         helpers.bookmarked? @document
       end
 
+      def bookmark_icon
+        icon = helpers.blacklight_icon("bookmark")
+        return unless icon.include?('svg')
```

Comment:
Can you add a comment in the code explaining why we need to check the return value of the `blacklight_icon` helper?  Should we add a test for both paths here?

Reply:
I am not sure I understand the both paths question. What do you mean by that?

Reply:
In the rspec?

Reply:
Done